netlify.toml

```diff
@@ -6,9 +6,9 @@
   [headers.values]
     Content-Security-Policy = '''
       default-src 'self' https://*.netlify.app;
-      connect-src 'self' data: https://keybase.io https://lcd.nylira.net;
+      connect-src 'self' https://keybase.io https://lcd.nylira.net http://localhost:8010;
       img-src 'self' data: https://*.netlify.app https://s3.amazonaws.com;
-      script-src 'self' 'unsafe-eval' https://*.netlify.app;
+      script-src 'self' 'unsafe-inline' https://*.netlify.app;
```

Comment:
We can't add unsafe-inline. That's the point of a CSP right?

Reply:
can we remove `unsafe-eval` and `unsafe-inline`? 

just `self` means it would only allow js from explicitly stated URLs (do we need JS from anywhere that isn't just our own client code?)

Reply:
do we need `script-src` at all?

Reply:
So we can't remove unsafe-inline. Nuxt injects scripts into the page even in generate mode. Nuxt can provide hashes to the CSP but just in server mode. This sounds crazy but seems to be the reality.